src/controllers/auth.controller.js

```diff
@@ -0,0 +1,56 @@
+import { AuthService } from "../services/auth.service.js";
+
+export class AuthController {
+  static async createAccount(req, res) {
+    try {
+      const { user_name, user_email, user_password } = req.body;
+      const newAccount = await AuthService.register({
+        name: user_name,
+        email: user_email,
+        password: user_password,
+      });
+      console.log("controller: ", newAccount);
```

Comment:
```suggestion
      console.log("[AuthController.createAccount]: ", account);
```

as long as you don't have a logger, we write like this, then the part in [ ] is removed